cmd/serve.go

```diff
@@ -80,11 +79,16 @@ func initRouter(
 		rootLogger.Info("lineage build complete")
 	}()
 
-	pgClient := initPostgres(config)
-	tagRepository := postgres.NewTagRepository(pgClient)
-	tagTemplateService := tag.NewTemplateService(
-		postgres.NewTemplateRepository(pgClient),
-	)
+	pgClient := initPostgres(rootLogger.WithField("reporter", "postgres"), config)
+	tagRepository, err := postgres.NewTagRepository(pgClient)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
should we somehow format/wrap the error here for easier debugging?
e.g. `error initiating tag repository: %w`

Reply:
yeah, can do that.